config/config.go

```diff
@@ -221,6 +222,19 @@ func OptionExperimental(exp bool) Option {
 	}
 }
 
+// OptionNetworkControlPlaneMTU function returns an option setter for control plane MTU
+func OptionNetworkControlPlaneMTU(exp int) Option {
+	return func(c *Config) {
+		logrus.Debugf("Network Control Plane MTU: %d", exp)
+		if exp < 1500 {
+			// if exp == 0 the value won't be used
+			logrus.Warnf("Received a Network Control Plane MTU of %d, this value is very LOW,",
+				"the database can misbehave, 0 value will be ignored", exp)
```

Comment:
ok for `low` also if I wanted to convey fear :D
ok for `network control-plane`
I put the 0 value because here https://github.com/docker/libnetwork/pull/1839/files#diff-2fe34e1b6f7f9aece0af74b1635502eeR217 it is possible to see 0 if the user set it so.